app/views/log_events/index.html.haml

```diff
@@ -0,0 +1,51 @@
+= content_for :h1 do
+  = text("pages.global_settings")
+
+= content_for :sidebar do
+  = render("admin/shared/sidenav_global", sidenav_tab: "log_events")
+
+%h2= text("pages.event_log")
+
+= form_tag(log_events_path, method: :get) do
+
+  = label_tag(text(:event_filter))
+
+  - option_list = LogEvent::LogEventSearcher::ALLOWED_EVENTS.map { |event| [text(event), event] }
+
+  = select_tag(:events,
+    options_for_select(option_list, params[:events]),
+    multiple: true,
+    class: "js--chosen",
+    data: { placeholder: text(:event_placeholder) })
+
+  = label_tag(text(:start_date))
+  = text_field_tag(:start_date, params[:start_date], class: :datepicker__data)
+
+  = label_tag(text(:end_date))
+  = text_field_tag(:end_date, params[:end_date], class: :datepicker__data)
+
+  = label_tag(text(:query))
```

Comment:
Can we improve the styling of the form at all? I don't want to spend a lot of time on it, but the fact that the button and the text input aren't aligned bugs me a little. 

I'm also thinking the query should be first item, and then the type would be the least important filter. I'm guessing (I could of course be wrong) that the most common use case will be to see the history of a single account.

![screen shot 2017-07-27 at 5 17 26 pm](https://user-images.githubusercontent.com/1099111/28694513-94af0da4-72ef-11e7-84b2-d1ce9e1d9a0f.png)